src/styles/index.scss

```diff
@@ -4,0 +1,7 @@
+@import './utils/variables';
+@import 'blocks/page';
+@import 'blocks/header';
+@import 'blocks/nav';
+@import 'blocks/main';
+@import 'blocks/card';
+@import 'blocks/stars';
```

Comment:
Ensure that all the imported block SCSS files (`page`, `header`, `nav`, `main`, `card`, `stars`) exist in the specified paths. This is crucial for the successful compilation of the SCSS to CSS.